lib/flow.js

```diff
@@ -2,8 +2,19 @@
 
 module.exports = (api) => {
 
+  api.metasync.flow = (
```

Comment:
What's the difference between `flow()` and `composition()`? Right now these functions are identical. And should `parallel()` and `sequential()` be updated to use `flow()` instead of `composition()`?